packages/i18n/tools/pot-to-php.js

```diff
@@ -47,8 +46,8 @@ function convertTranslationToPHP( translation, textdomain, context = '' ) {
 	let original = translation.msgid;
 	const comments = translation.comments;
 
-	if ( ! isEmpty( comments ) ) {
-		if ( ! isEmpty( comments.reference ) ) {
+	if ( Object.values( comments ).length ) {
+		if ( comments.reference ) {
```

Comment:
This now throws an error if `comments` is `null` or `undefined`

```log
TypeError: Cannot convert undefined or null to object
```

Reply:
Nice catch. Fix suggested in #45414.

Test instructions to reproduce would be really helpful to ensure we fixed it properly @manzoorwanijk .